server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOAWSTest.java

```diff
@@ -73,6 +75,8 @@ void setUp(DockerAwsS3Container dockerAwsS3) {
     void tearDown() {
         blobStore.deleteContainer(defaultBucketName.asString());
         blobStore.deleteContainer(CUSTOM.asString());
+        blobStore.deleteContainer(ObjectStorageBlobsDAOWithPrefixContract.DEFAULT_BUCKET_NAME.asString());
+        blobStore.deleteContainer(ObjectStorageBlobsDAOWithPrefixContract.RESOLVED_BUCKET_NAME.asString());
```

Comment:
Please use delete Bucket and remove this lower level, of-contract `deleteContainer` method